packages/server/graphql/public/types/NotifyResponseMentioned.ts

```diff
@@ -0,0 +1,20 @@
+import NotificationResponseMentioned from '../../../database/types/NotificationResponseMentioned'
+import {NotifyResponseMentionedResolvers, ResolversTypes} from '../resolverTypes'
+
+export type NotifyResponseMentionedSource = Pick<
```

Comment:
-1 can remove & in codegen.json reference the database type directly like https://github.com/ParabolInc/parabol/blob/6fae361e2fa77b7c21e33100c60621303b236ac0/codegen.json#L50

unless i'm missing something?